### PR TITLE
SAK-49599 Portal: Fix lost focus on eid and pw fields

### PR DESCRIPTION
**Pull request by stetsche**
The added if clause clause will prevent loosing the focus from the login fields to the focus on the portal wrapper.

Removing the "focus password field" code will prevent the username field from becoming focus while typing the password. This piece of code was already inactive due to the code focusing on the portal wrapper.

https://sakaiproject.atlassian.net/browse/SAK-49599

### CODE REVIEW

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm [hunk 1]
```diff
@@ -164,8 +164,10 @@
 
         // put the focus back on the portal wrapper so that keyboard navigation works
         setTimeout(function() {
-
-            document.querySelector('.Mrphs-portalWrapper').focus();
+            // Do not mess with the focus if the login fields are in focus right now
```
Comment (kunaljaykam) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm:167]: If the comment is required here, Can we do something like this?
```suggestion
            // Preserve focus on login fields if they're currently active
```

**Review by kunaljaykam — COMMENTED** [review]
Looks good to me,  just a small suggestion.

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm (hunk 1 above)
Comment (stetsche) [reply portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm:167]: Of course :)

**Review by kunaljaykam — APPROVED** [review]
Thanks @stetsche

**Comment by stetsche** [comment]
Thank YOU @kunaljaykam  :)

**Pull request merged**